modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/QuantileCalculationMethod.java

```diff
@@ -103,37 +63,16 @@ public double quantileWithExtrapolationFromUnsorted(double level, DoubleArray sa
    * <p>
    * If index value computed from the level is outside of the sample data range, the nearest data point is used, i.e., 
    * expected short fall is computed with flat extrapolation.
-   * Thus this is coherent to {@link #quantileWithExtrapolationFromSorted(double, DoubleArray)}.
-   * <p> 
-   * The sample observations are sorted from the smallest to the largest. 
-   * 
-   * @param level  the quantile level
-   * @param sortedSample  the sample observations
-   * @return the quantile estimation
-   */
-  public double expectedShortfallFromSorted(double level, DoubleArray sortedSample) {
-    return expectedShortfall(level, sortedSample);
-  }
-
-  /**
-   * Compute the expected shortfall.
-   * <p>
-   * The quantile level is in decimal, i.e. 99% = 0.99 and 0 < level < 1 should be satisfied.
-   * This is measured from the bottom, that is, Thus the expected shortfall with the level 99% corresponds to 
-   * the average of the smallest 99% of the observations.
-   * <p>
-   * If index value computed from the level is outside of the sample data range, the nearest data point is used, i.e., 
-   * expected short fall is computed with flat extrapolation.
    * Thus this is coherent to {@link #quantileWithExtrapolationFromUnsorted(double, DoubleArray)}.
    * <p> 
-   * The sample observations are supposed to be unsorted, the first step is to sort the data.
+   * The sample observations are sorted from the smallest to the largest. 
```

Comment:
Please add javadoc mentioning that `indices` in `QuantileResult` are based on the original unsorted array.